sample.py

```diff
@@ -1,95 +1,145 @@
 from typedb.driver import TypeDB, SessionType, TransactionType, TypeDBOptions
-from datetime import datetime
-DB_NAME = "iam"
+import setup
+
+DB_NAME = "sample_app"
 SERVER_ADDR = "127.0.0.1:1729"
 
-print("IAM Sample App")
-
-print("Attempting to connect to a TypeDB Core server: ", SERVER_ADDR)
-with TypeDB.core_driver(SERVER_ADDR) as driver:  # Connect to TypeDB Core server
-    if driver.databases.contains(DB_NAME):
-        print("Found a pre-existing database! Re-creating with the default schema and data...")
-        driver.databases.get(DB_NAME).delete()
-    driver.databases.create(DB_NAME)
-    if driver.databases.contains(DB_NAME):
-        print("Empty database created.")
-    print("Opening a Schema session to define a schema.")
-    with driver.session(DB_NAME, SessionType.SCHEMA) as session:
-        with session.transaction(TransactionType.WRITE) as transaction:
-            with open('iam-schema.tql', 'r') as data:
-                define_query = data.read()
-            transaction.query.define(define_query)
-            transaction.commit()
-    print("Opening a Data session to insert data.")
-    with driver.session(DB_NAME, SessionType.DATA) as session:
-        with session.transaction(TransactionType.WRITE) as transaction:
-            with open('iam-data-single-query.tql', 'r') as data:
-                insert_query = data.read()
-            transaction.query.insert(insert_query)
-            transaction.commit()
-        print("Testing the new database.")
-        with session.transaction(TransactionType.READ) as transaction:  # Re-using a session to open a new transaction
-            test_query = "match $u isa user; get $u; count;"
-            response = transaction.query.get_aggregate(test_query)
-            result = response.resolve().as_value().as_long()
-            if result == 3:
-                print("Database setup complete. Test passed.")
+
+def fetch_all_users(driver):
+    print("\nRequest #1: Fetch all users as JSON objects with full names and emails")
```

Comment:
The pattern of defining a somewhat re-usable function, but printing this very specific message in it is a little unintuitive. Consider moving the message into `main`?